README.md

```diff
@@ -51,6 +51,15 @@ If you want to work on improving the components it’s best to run the latest co
     - In the repository of an app do `npm link @nextcloud/vue` (you need to re-link any time you do `npm ci` in the app)
 4. Then build the app with: `npm run build` (or watch for changes with `npm run watch`)
 
+## Releasing a new version
+
+1. Checkout latest master
+2. Bump version in `package.json` and update `package-lock.json`
```

Comment:
🚨🚨🚨
```suggestion
2. use `npm version` to bump the version number
```

Reply:
ah that updates package.json?

Reply:
Yes, it also automatically commits it and creates a git tag